### DISCUSSION TITLE
CROS error in breeze

### DISCUSSION THREAD

**Discussion by dheerajKsharma** [breezefront/community#180] [1. Help]
Hi,
I am facing CROS error when hitting a 3rd party api for fetching data. If I run same page in LUMA it's works well and I can get the data.

![image](https://github.com/breezefront/community/assets/132253509/3d2a5e9b-1742-47c5-8527-60457df9402c)

Please let me know If any solution is around.
Thanks

**Comment by vovayatsyuk** [comment]
Please provide URLs for both working and not working sites.

**Reply by dheerajKsharma** [reply]
Hi @vovayatsyuk ,
Actually currently we are doing development in local only. We are fetching data from this URL: https://plugin-test.heritage-digital.co.uk/api/framed/getmountgroups?siteid=2&galleryid=2&reference=207497&producttype=Framed

**Reply by vovayatsyuk** [reply]
Here is a workaround: 

1. Apply [this patch](https://github.com/breezefront/module-breeze/commit/1ada8caada6a70854cf8ff226e3f375635aedd9a)
2. Send request as follows:

    ```js
    $.ajax(url, {
        headers: {
            'X-Requested-With': false
        }
    })
    ```

**Reply by dheerajKsharma** [reply]
Perfect! Worked like a charm :) Thanks again.

**Reply by dheerajKsharma** [reply]
I have sent it to server, But it's not working as expected. It''s still giving errors in console. 
Can you please have a look: https://artuk.wcltest.com/virgin-and-child-207017.html

**Reply by vovayatsyuk** [reply]
Looks like an error in the server config. If you read the error in console, you'll find this:

```
The 'Access-Control-Allow-Origin' header contains multiple values 'https://artuk.wcltest.com, *', but only one is allowed.
```
